File: Auto.py
from pytiktok import TikTok

# Initialize TikTok object and set the access token
tiktok = TikTok(access_token='YOUR_ACCESS_TOKEN')

# prepare the video and metadata to be uploaded
video = open('path/to/video.mp4', 'rb')
caption = "Here is my awesome video!"
hashtags = ["awesome", "cool", "video"]

# upload the video
result = tiktok.upload_video(video=video, caption=caption, hashtags=hashtags)

print(result)

File: Auto.rs
use tiktok::prelude::*;

// Create a TikTok client with the access token
let client = TikTok::with_access_token("YOUR_ACCESS_TOKEN");

// Prepare the video file and metadata for the upload
let video = File::open("path/to/video.mp4")?;
let caption = "Here is my awesome video!";
let hashtags = vec!["awesome", "cool", "video"];

// Upload the video
let res = client.upload_video(video, caption, hashtags)?;

println!("{:?}", res);
